misc/scripts/mapcreation/brouter.sql

```diff
@@ -598,25 +595,84 @@ ORDER BY
 --
 SELECT
     y.osm_id losmid,
-    CASE WHEN y.green_factor < 0.1 THEN
+    CASE WHEN y.green_factor < 0.5 THEN
         NULL
-    WHEN y.green_factor < 0.2 THEN
+    WHEN y.green_factor < 0.6 THEN
         '1'
-    WHEN y.green_factor < 0.4 THEN
+    WHEN y.green_factor < 0.7 THEN
         '2'
-    WHEN y.green_factor < 0.6 THEN
-        '3'
     WHEN y.green_factor < 0.8 THEN
+        '3'
+    WHEN y.green_factor < 0.9 THEN
         '4'
     WHEN y.green_factor < 0.98 THEN
         '5'
     ELSE
         '6'
+    END AS green_class INTO TABLE green_tags
+FROM
+    green_tmp y
+WHERE
+    y.green_factor > 0.5;
+
+SELECT
+    count(*)
+FROM
+    green_tags;
+
+SELECT
+    green_class,
+    count(*)
+FROM
+    green_tags
+GROUP BY
+    green_class
+ORDER BY
+    green_class;
+
+SELECT
+    now();
+
+-- create tags for forest
+SELECT
+    m.osm_id,
+    m.highway,
+    st_area (st_intersection (m.way, ST_Union (q.way))) / st_area (m.way) AS forest_factor INTO TABLE forest_tmp
+FROM
+    osm_line_buf_50 AS m
+    INNER JOIN osm_poly_no_buf AS q ON ST_Intersects (m.way, q.way)
+WHERE
+    m.highway IS NOT NULL
+    AND q.landuse IN ('forest')
+GROUP BY
+    m.osm_id,
+    m.highway,
+    m.way
+ORDER BY
+    forest_factor DESC;
+
+--
+SELECT
+    y.osm_id losmid,
+    CASE WHEN y.forest_factor < 0.5 THEN
+        NULL
```

Comment:
Is this line needed?

Reply:
The same question applies to the current upstream code: https://github.com/abrensch/brouter/blob/76265e7713ad203ee6679a810121d2bd22f78074/misc/scripts/mapcreation/brouter.sql#L601-L602

Reply:
You are right, it works just fine without these lines.